### ISSUE TITLE
What does the metric "impressions" describe?

### ISSUE THREAD

**Issue by trieloff** [adobe/xdm#244]
https://github.com/adobe/xdm/blob/592130cfc939f0d43ccc7632b56661be00211f44/schemas/data/impressions.schema.json#L14

**Comment by cdegroot-adobe** [comment]
@trieloff I do not believe those schema files that each describe an individual metric are used. Can we just delete them? If they have value please explain how.

The metrics are described in the schemas/data/metrics.schema.json

**Comment by trieloff** [comment]
The description within `schemas/data/metrics.schema.json` is nice to have, but does not get picked up by any tooling. As soon as you have a `$ref` somewhere, the contents of the `$ref` are replacing everything that's inside the object that contains the `$ref`.

The individual descriptions that you suggested to delete, are shown here: https://github.com/adobe/xdm/blob/master/docs/reference/data/impressions.schema.md

**Comment by kstreeter** [comment]
I think this gets fixed by #280

**Comment by trieloff** [comment]
No, #280 does not touch `xdm/schemas/data/impressions.schema.json`
